README.md

```diff
@@ -409,6 +419,22 @@ exports.handler = (event, context, callback) => {
 };
 ```
 
+## HTTP Exception handling
+If the logger did not receive a successful response from the server when tried to send a log,
+it will retain the logs in its buffer and will try to flush it with the next request.
+The flush limit of the line that comes after a failed one will be reset to the backoff period.
+The failed messages retention byte size is limited. The limit and back off
+period are configurable via options:
+
+``` javascript
+const options = {
+    failedBufRetentionLimit: 10000000
+    backOffPeriod: 3000
```

Comment:
The term "backoff" is I think usually used when there's something like an exponential backoff; if it's always fixed, I think it's more common to use "retry". So I'd prefer `retryTimeout`